index.html

```diff
@@ -0,0 +1,144 @@
+        <li class="menu-desktop-list-itens color-one">Join</li>
+        <li class="menu-desktop-list-itens color-one">Sponsor</li>
+        <li class="menu-desktop-list-itens color-one">News</li>
+      </ul>
+    </div>
+    <section class="header-section">
+      <p class="color-two event-call">"Hello game lovers!"</p>
+      <h1 class="event-name">BRASIL GAME SHOW 2022</h1>
+      <p class="event-desc">An event for those who are in love with everything about the geek world. Here you
+        can find cosplayers, influencers, competitions and more.</p>
+      <h2 class="event-date">2022.10.06(Thu) ~ 12(Mon)</h2>
+      <h3 class="event-place">@EXPO CENTER NORTE, SP</h3>
+    </section>
+  </header>
+  <main class="main">
+    <section class="program-section">
+      <h1 class="color-three program-section-title">Main Progam</h1>
+      <hr class="section-line">
+      <div class="desktop-cards">
+        <div class="program-card">
+         <i class="fa-solid fa-message card-icons"></i>
+         <h3 class="program-card-name color-two">
+           BGS Talks
+         </h3>
+         <p class="program-card-desc color-three">
+           The visitors will have the oportunity to watch amazing presentation by international guests and influencers.
+         </p>
+        </div>
+        <div class="program-card">
+         <i class="fa-solid fa-masks-theater card-icons"></i>
+         <h3 class="program-card-name color-two unic-font">
+           BGS Cosplay
+         </h3>
+         <p class="program-card-desc unic-font color-three">
+           The BGS welcomes a thousand of cosplayers who shows real masterpieces and theatrical performances.
+         </p>
+        </div>
+        <div class="program-card">
+         <i class="fa-solid fa-user card-icons"></i>
+         <h3 class="program-card-name color-two unic-font">
+           BGS Meet & Greet
+         </h3>
+         <p class="program-card-desc unic-font color-three">
+           On the stage at the Meet & Greet have passed several personalaties from games, this is your chance to see them.
+         </p>
+        </div>
+        <div class="program-card">
+         <i class="fa-regular fa-circle-play card-icons"></i>
+         <h3 class="program-card-name color-two unic-font">
+           400 Exhibitors
+         </h3>
+         <p class="program-card-desc unic-font color-three">
+           With a record of brands present, BGS2022 will be full of attractions. You can't miss it!
+         </p>
+        </div>
+        <div class="program-card">
+         <i class="fa-solid fa-gamepad card-icons"></i>
+         <h3 class="program-card-name color-two unic-font">
+           BGS Jam
+         </h3>
+         <p class="program-card-desc unic-font color-three">
+           BGS Jam is pure emotion!In 48 hours, teams made up of students try yo develop the best game to win the prize.
+         </p>
```

Comment:
- [ ] I would suggest you please use the padding from all sides and there is a white background on each card. please implement it
### Your design
![his](https://user-images.githubusercontent.com/81727988/191887569-0dac99f7-ec16-473a-ab17-91fd3fba808e.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/191887604-5f9c2e55-6c26-419b-95c8-c9c176336cbe.png)